### DISCUSSION TITLE
SNES Romhacks

### DISCUSSION THREAD

**Discussion by MrMeaker** [eduardofilo/RG350_adam_image#303] [General]
I've just started getting into  some SMW ROMHacks, and although the hacks I have work fine on other emulators, and on the stock firmware of my RG350M, they will not open with the Adam Image. I've tried using all 3 emulators for the SNES, but they all just go to a black screen and hang there. Any ideas?

**Comment by a-dekker** [comment]
My success rate was not that impressive either. Tried 3 random patches, only "New Super Mario World 2: Around The World" worked, using PocketSNES, no luck with the retroarch cores. Seems they have a hard time handling patched roms.
